files/package.json

```diff
@@ -25,7 +25,7 @@
     "lint:js:fix": "eslint . --fix",<% if (typescript) { %>
     "lint:types": "glint",<% } %>
     "start": "vite dev",
-    "test": "vite build --mode=development && testem --file testem.cjs ci",
+    "test": "vite build --mode=development && testem --file testem.cjs ci --port 0",
```

Comment:
so... I don't think that testem supports the `--port 0` way to pick the next open port 🤔 can we just remove it? 

```suggestion
    "test": "vite build --mode=development && testem --file testem.cjs ci",
```

Reply:
I'll ask the testem folks, but this shouldn't block this pr, since it's the better behavior

Reply:
https://github.com/testem/testem/issues/1901

Reply:
I dug in a bit deeper:
- it's an express server: https://github.com/testem/testem/blob/27ba5a50b16ff0f59fc13c3a559494658ff9cd5b/lib/server/index.js#L87

- here is where it starts listening: https://github.com/testem/testem/blob/27ba5a50b16ff0f59fc13c3a559494658ff9cd5b/lib/server/index.js#L65

which lead me to the docs: https://expressjs.com/en/api.html#app.listen

so, TIL, this behavior is pretty engrained in all of node!

![image](https://github.com/user-attachments/assets/4fcf69f0-b174-4b50-abdb-efd3d5944e81)

we keep it